scripts/setup_mahara.sh

```diff
@@ -0,0 +1,510 @@
+# Custom Script for Linux for Mahara/Azure
```

Comment:
Why is this PR adding a Mahara script. This repo is for Moodle and so this seems out of place. Was this accidentally committed or is there an objective here. If so what issue does this relate to?

Reply:
No, appears to have been added incidentally.

Reply:
:-) easily done. You may be interested in an open issue to separate out the LAMP deployment parts of this template so that it will be reusable across different LAMP applications. See #101